src/components/donationCard/DonationCard.js

```diff
@@ -0,0 +1,160 @@
+        { donationAmount: data.amount, comment: data.comment }
+      );
+      window.location.assign(response.data.payment._links.checkout.href);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return ReactDom.createPortal(
+    <div className="popup">
+      <div className="donation-card">
+        <div className="donation-card__header">
+          <h3 className="donation-card__title">{props.titleHeader}</h3>
+          <p className="donation-card__text donation-card__text--white">
+            {props.by}
+          </p>
+        </div>
+        <form onSubmit={handleSubmit(onSubmit)} className="donation-card__main">
+          <div className="donation-card__tags">
+            {props.tags.map((tag) => {
+              return (
+                <p
+                  key={tag.name}
+                  className="donation-card__tags donation-card__tags--css"
+                >
+                  {tag.name}
+                </p>
+              );
+            })}
+          </div>
+          <p className="donation-card__text donation-card__text--black">
+            {props.description}
+          </p>
+          <h4>How much do you want to donate?</h4>
+          <div className="donation-card__amount">
+            <aside className="donation-card__amount-display donation-card__amount-display--white">
+              €25
+            </aside>
+            <aside className="donation-card__amount-inputs">
+              {[1, 5, 10, 25, 50, 100, 250].map((amount) => {
+                return (
+                  <button key={amount} className="donation-card__amount-btn">
+                    € {amount}
+                  </button>
+                );
+              })}
+              <label className="form__label--input" htmlFor="amount">
+                <input
+                  className="form__input"
+                  type="number"
+                  placeholder="Enter an amount"
+                  {...register("amount", { required: true })}
+                />
+              </label>
+            </aside>
+          </div>
+          <h5>Leave a comment with your donation (optional)</h5>
+          <div className="donation-card__comment">
+            <label className="form__label" htmlFor="comment">
+              <textarea
+                className="form__input"
+                placeholder="Leave your comment here"
+                {...register("comment", { maxLength: 255 })}
+              />
+            </label>
+            <p className="donation-card__charleft">0/255 characters</p>
+          </div>
+          <div className="donation-card__btn">
+            <Button
+              type="submit"
+              icon={
+                <FaHandHoldingHeart
+                  style={{
+                    color: "white",
+                    fontSize: "0,5em",
+                    marginRight: "0.5rem",
+                  }}
+                />
+              }
+              text={`support ${props.by} with €€€€`}
```

Comment:
The amount on this button is not dynamic (and the text is slightly off) as it contains: `by: ` and also the lastname

Made a card for this in trello
<img width="190" alt="Screenshot 2021-07-19 at 11 32 05" src="https://user-images.githubusercontent.com/20372832/126138420-87b5fff9-4060-4f5c-9ed1-b96c85dcfcc9.png">